### ISSUE TITLE
[Higher Level Review]: Error in ARIA label on Review and Submit page

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#83691]
## What happened?

**Describe what went wrong**:

When you get to the Review and Submit page, in the accordion with the "Disagreement with [issue] decision on [date]", the edit button has broken `aria-label`.

**Add the steps to reproduce the bug**:

1. Log in with a test user account. For example: `vetsgov+test236@id.me`.
2. Proceed to the Review and Submit page after filling out the form.
3. Expand all the accordions.
4. Turn on a screen reader or open HTML source.
4. Find the "Disagreement with [issue] decision on [date]" button and note the wrong `aria-label`. Says: "Edit [object Object]".

It should say "Edit Disagreement with [issue] decision on [date]" This is a WCAG violation of [2.5.3 Label in Name](https://www.w3.org/TR/WCAG22/#label-in-name).

**Add a screen recording or photo**:

<img width="668" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOHI5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0b66981d429f41986abc981dd9608cbba1e85ced/Screenshot%202024-05-20%20at%205.20.35%E2%80%AFPM.png" alt="Review and Submit page with red arrows pointing to broken edit button and its aria-label in HTML." />

## How severe is the problem?

- [ ] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [x] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [ ] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. _When a user does X, the **visual** result is y._
2. _When a user does X, the **backend** result is y._
3. _When a user does X, the **event logging** result is y._

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place

**Comment by Mottie** [comment]
This work has been merged in and ready for validation in staging.

**Comment by davidakennedy** [comment]
Tested this, and see the proper label in HTML. Works as expected.

![Review and Submit page with red arrow pointing to its aria-label in HTML.](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1473618/25ac69ad-cc6c-4f44-bc9e-843ba920bd04)
